habitat-lab/habitat/articulated_agent_controllers/humanoid_seq_pose_controller.py

```diff
@@ -58,6 +59,15 @@ def reset(self, base_transformation: mn.Matrix4) -> None:
 
         # The first pose of the motion file will be set at base_transformation
         self.ref_pose = base_transformation
+        self.snap_offset = mn.Matrix4()
+        self.calculate_pose()
+
+    def snap_to(self, base_transformation: mn.Matrix4) -> None:
+        """Snaps the current pose to the base transformation, so that the rest of poses are relative to this one"""
```

Comment:
Perhaps it would be clearer if we renamed this function `apply_base_transformation`.

Reply:
addressed